cgt/core.py

```diff
@@ -1093,13 +1093,13 @@ def _nu_iceil(x,out=None):
     if out is None:
         return np.ceil(x)
     else:
-        np.ceil(x,out)
+        out[...] = np.ceil(x)
```

Comment:
What numpy version do you use? I don't see why that fix is needed:
```
In [1]: import numpy as np

In [2]: np.__version__
Out[2]: '1.8.2'

In [3]: x = np.random.randn(10, 10).astype(np.float32)

In [4]: y = np.zeros((10, 10), dtype=np.int64)

In [5]: x.dtype, y.dtype
Out[5]: (dtype('float32'), dtype('int64'))

In [7]: _ = np.ceil(x, y)

In [8]: x.dtype, y.dtype
Out[8]: (dtype('float32'), dtype('int64'))
```

The previous version is better because it doesn't need to assign a temporary variable and copy the result into `out`.